Mathlib/MeasureTheory/Measure/LevyProkhorovMetric.lean

```diff
@@ -198,6 +194,24 @@ lemma levyProkhorovDist_triangle [OpensMeasurableSpace Ω] (μ ν κ : Measure 
 when they are to be equipped with the Lévy-Prokhorov distance. -/
 def LevyProkhorov (α : Type*) := α
 
+/-- Coercion from the type synonym `LevyProkhorov (ProbabilityMeasure Ω)`
+to `ProbabilityMeasure Ω`. -/
```

Comment:
If you want this, and the other three `def`s below to be coercions, then you need to do two things:

1. Define appropriate `Coe` instances for each. (this allows Lean to look for the coercion during elaboration, so that you don't have to manually insert it)
2. add the `@[coe]` attribute, which allows Lean to display a `↑` in the infoview and makes `norm_cast` aware that this is a coercion for its purposes.

However, that being said, I tend to think that you don't want *any* of these. Instead, it seems like you want a "parameterized type synonym", much like we have for `WithLp` or `Lex`. So I think you just want a *single* `def`, and it should be an `Equiv`, not a plain function. That is:
```lean
-- maybe you could make `α` implicit?
def LevyProkhorov.equiv (α : Type*) : LevyProkhorov α ≃ α where
  ...
```

Reply:
Thanks! I did this and it looks cleaner indeed --- maybe a little less descriptive as naming, but much better that the "coercion" is not special-cased to both cases of interest.

Reply:
I noticed that I should still update naming of the lemmas to refer to this equiv, so I marked as "awaiting author" again until then...